### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by Acid-Override**
Playing around with github and learning how to submit PR.

### CODE REVIEW

**Comment by Acid-Override** [comment]
I read through the log file and don't understand why it failed to build?

File: index.rst [hunk 1]
```diff
@@ -5,10 +5,17 @@ Welcome to the official documentation of the UBports project!
...
+Documentation creation is being provided by Read the Docs (https://readthedocs.org/ ) which is an open source (MIT-licensed) project.  The guide is divided into general chapters to make reading easier. Docs are rendered as HTML and PDF and full-text search is available.  If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
+
+This guide is constantly being improved. Although we attempt to provide you with a comprehensive collection of helpful information, we know we won't answer all of your questions here. We will keep adding more information to make things more helpful. 
+
+Ubuntu Touch is community supported and depends on users like you to contribute to development, support, and operations. You can learn more about how to :doc:`contribute <contribute/documentation>`to our projects. Thanks so much to our wonderful community team who helps us run the site. Ubuntu Touch wouldn't be possible without them. Please consider joining today, as we always need more contributors to the project.
```
Comment (NeoTheThird) [inline index.rst]: You missed a space after the link

Comment (Acid-Override) [reply index.rst]: OK.. let me try again.

File: index.rst [hunk 2]
```diff
@@ -5,10 +5,17 @@ Welcome to the official documentation of the UBports project!
...
+This guide is constantly being improved. Although we attempt to provide you with a comprehensive collection of helpful information, we know we won't answer all of your questions here. We will keep adding more information to make things more helpful. 
+
+Ubuntu Touch is community supported and depends on users like you to contribute to development, support, and operations. You can learn more about how to :doc:`contribute <contribute/documentation>` to our projects. Thanks so much to our wonderful community team who helps us run the site. Ubuntu Touch wouldn't be possible without them. Please consider joining today, as we always need more contributors to the project.                 
+
+
+Thank you for taking the time to read the UBports Ubuntu Touch guide.  
+    -- The UBports Documentation team  
+
 
-.. note::
```
Comment (NeoTheThird) [inline index.rst]: Imho this should stay in. Any particular reason why you removed it? I'd vote to keep it unitl the documentation is more finished.

File: index.rst [hunk 3]
```diff
@@ -5,10 +5,17 @@ Welcome to the official documentation of the UBports project!
 
 UBports develops the mobile phone operating system Ubuntu Touch. Ubuntu Touch is a mobile operating system focused on ease of use, privacy, and convergence.
 
-On this website you find instructions how to install Ubuntu Touch on your mobile phone, user guides and detailed documentation on all system components. If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
```
Comment (NeoTheThird) [inline index.rst]: Why remove this? Please also see the comment son #27, since that PR touches the same files and is still open as well.

File: index.rst [hunk 4]
```diff
@@ -5,10 +5,17 @@ Welcome to the official documentation of the UBports project!
...
+Documentation creation is being provided by Read the Docs (https://readthedocs.org/ ) which is an open source (MIT-licensed) project.  The guide is divided into general chapters to make reading easier. Docs are rendered as HTML and PDF and full-text search is available.  If this is your first time here, please consider reading our :doc:`introduction <about/introduction>`.
```
Comment (NeoTheThird) [inline index.rst]: You're repeating content that's already covered on about/introduction. If you want to move/remove/replace a page, please do it in one pull request, because we won't merge unfinished pages in master.

File: index.rst (hunk 1 above)
Comment (NeoTheThird) [reply index.rst]: That worked :+1:
